src/locales/cy/translation.json

```diff
@@ -1053,29 +1044,29 @@
       }
     },
     "pageIpvIdentityDocumentTypes": {
-      "title": "Types of photo ID",
-      "header": "Types of photo ID",
+      "title": "Mathau o ID gyda llun",
+      "header": "Mathau o ID gyda llun",
       "content": {
-        "subHeading": "UK photocard driving licence",
-        "paragraph": "You can use a UK driving licence if it:",
-        "drivingLicenceRequirements": ["has your photo", "has not expired"],
-        "paragraph1": "It can be a full or provisional UK driving licence.",
-        "subHeading2": "UK passport",
-        "paragraph3": "You can use any UK passport if it has not expired.",
-        "subHeading3": "Non-UK passport with a biometric chip",
-        "paragraph4": "You can use a non-UK passport if it:",
+        "subHeading": "Trwydded yrru cerdyn-llun y DU",
+        "paragraph": "Gallwch ddefnyddio trwydded yrru'r DU os yw:",
+        "drivingLicenceRequirements": ["gyda'ch llun", "heb ddod i ben"],
+        "paragraph1": "Gall fod yn drwydded yrru lawn neu dros dro y DU.",
+        "subHeading2": "Pasbort y DU",
+        "paragraph3": "Gallwch ddefnyddio unrhyw pasbort y DU os yw heb ddod i ben.",
+        "subHeading3": "Pasbort o’r tu allan i’r DU gyda sglodyn biometrig",
+        "paragraph4": "Gallwch ddefnyddio pasbort o’r tu allan i’r DU os yw:",
         "nonUkPassportRequirements": [
-          "has a biometric chip",
-          "has not expired"
+          "gyda sglodyn biometrig",
+          "heb ddod i ben"
         ],
-        "paragraph5": "To check if your passport has a biometric chip, look for the rectangular biometric chip symbol on the front cover.",
-        "imgAltText": "The front cover of a passport with the biometric chip symbol. The symbol is rectangular with a circle in the middle.",
-        "subHeading4": "UK biometric residence permit (BRP)",
-        "paragraph6": "You can use a UK biometric resident permit (BRP) that has not expired. You might have a BRP if you have a visa for longer than 6 months or indefinite leave to remain.",
-        "subHeading5": "UK biometric residence card (BRC)",
-        "paragraph7": "You can use a BRC that has not expired. You might have a BRC if you’re in the UK as a family member of a citizen from a European Economic Area (EEA) country.",
-        "subHeading6": "UK Frontier Worker permit (FWP)",
-        "paragraph8": "You can use an FWP that has not expired. You might have an FWP if you work in the UK but live in another country."
+        "paragraph5": "I wirio os oes gan eich pasbort sglodyn biometrig, edrychwch am y symbol sglodyn biometrig petryal ar y clawr blaen.",
+        "imgAltText": "Clawr blaen pasbort gyda'r symbol sglodyn biometrig. Mae'r symbol yn betryal gyda chylch yn y canol.",
```

Comment:
Was you able to confirm if we needed the special character, I see it used elsewhere now `’` for welsh translations. If so it should be added here?

Reply:
Also this text isn't identical :( there's `(FWP)` at the end that doesn't appear in the existing component - probably an oversight by UCD

Reply:
I have raised this with UCD, they don't have an immediate answer though

Reply:
UCD have said that we can add `(FWP)` everywhere